pkg/libxdploader/libxdp_loader.go

```diff
@@ -0,0 +1,79 @@
+package libxdploader
```

Comment:
What about moving this into its own upstream repository and pulling it in, with the goal to make this a go goto library for libxdp Overkill?